genestack_client/metainfo_scalar_values.py

```diff
@@ -38,7 +38,7 @@ def _xstr(arg):
         :return: string representation of the object
         :rtype: str
         """
-        return str(arg) if arg is not None else None
+        return unicode(arg, 'utf-8').encode('utf-8', 'ignore') if arg is not None else None
```

Comment:
Now this method accepts *only* `str` arguments.  If it is intended, docstring should be fixed (probably with exception handling).  But I personally would like to see a more thorough attempt to "cast" `object` to `str`.

Reply:
I've given it a bit more of thought: your changes are basically a no-op.  If `arg = None`, then you return `None`.  If it's a `str`, you _assume_ it's UTF8-encoded, decode it to `unicode`, then encode back to UTF8.  What's the point?

Reply:
I see you point. Updated.